From f0d758dff4cdddf11535bfb96bc2a15d65cfe80f Mon Sep 17 00:00:00 2001
From: Colin Leroy <colin@colino.net>
Date: Fri, 21 Jul 2017 13:32:31 +0200
Subject: [PATCH 04/13] dav: Fix DAV implementation stripping spaces

This patch fixes DAV stripping leading and trailing spaces from server
responses. This fixes access to DAV files and directories which names start or
end with a space.

https://bugzilla.gnome.org/show_bug.cgi?id=785123
---
 daemon/gvfsbackendhttp.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/daemon/gvfsbackendhttp.c b/daemon/gvfsbackendhttp.c
index 6023adda..81ebe27e 100644
--- a/daemon/gvfsbackendhttp.c
+++ b/daemon/gvfsbackendhttp.c
@@ -100,7 +100,7 @@ http_path_get_basename (const char *path)
     return NULL;
 
   /* remove any leading slashes */
-  while (*path != '\0' && (*path == '/' || *path == ' '))
+  while (*path != '\0' && *path == '/')
     path++;
 
   len = strlen (path);
@@ -111,7 +111,7 @@ http_path_get_basename (const char *path)
   while (len)
     {
       char c = path[len - 1];
-      if (!g_ascii_isspace (c) && c != '/')
+      if (c != '/')
 	break;
 
       len--;
-- 
2.15.1

